changelogs/fragments/162-floating-ip-data-idempotency.yaml

```diff
@@ -0,0 +1,2 @@
+bugfixes:
+  - digital_ocean_floating_ip - make floating ip return data idempotent. (https://github.com/ansible-collections/community.digitalocean/pull/162)
```

Comment:
The period should be moved to after the reference URL, i.e.:

```yaml
bugfixes:
  - digital_ocean_floating_ip - make floating ip return data idempotent (https://github.com/ansible-collections/community.digitalocean/pull/162).
```

Reply:
Fixed it. Thanks for the guidance.